openreview/conference/templates/tabsConferenceWebfield.js

```diff
@@ -71,13 +71,7 @@ function load() {
       details: 'forumContent,writable'
     });
 
-    userGroupsP = Webfield.getAll('/groups', { regex: CONFERENCE_ID + '/.*', member: user.id, web: true })
-    .then(function(groups) {
-      return _.filter(
-        _.map(groups, function(g) { return g.id; }),
-        function(id) { return _.startsWith(id, CONFERENCE_ID); }
-      );
-    });
+    userGroupsP = Webfield.getAll('/groups', { regex: CONFERENCE_ID + '/.*', member: user.id, web: true });
```

Comment:
This still should have a `then` with a map that just returns group ids though, no?

Reply:
I'm doing that later

Reply:
line 165

Reply:
I see

Reply:
Is it ok? Can I merge this?

Reply:
Yeah, looks good to me